### PR TITLE
Fixed broken Comment-section

### PR DESCRIPTION
**Pull request by yash15601**
Fixes  https://github.com/CircuitVerse/CircuitVerse/issues/767

#### Describe the changes you have made in this pr -
Added .thread class which was missing and did some other changes

### Screenshots of the changes (If any) -
Before:
Screenshot (63)

After:
Screenshot (62)

### CODE REVIEW

**Comment by yash15601** [comment]
@tachyons  Views Please

**Comment by aayush-05** [comment]
Avoid fixing UI bugs for now other than the simulator. This page is being revamped anyways under new UI

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -1,4 +1,28 @@
 // All the custom class styles
+.thread {
+  display: block !important;
+}
+
+.thread_header {
+  display: block;
+}
+
+.thread_subscription {
```
[the remark this answers is not in the document]
Comment (tachyons) [reply app/assets/stylesheets/layout.scss]: @yash15601 Why this is resolved manually ?

**Comment by yash15601** [comment]
@aayush-05 Okay

**Comment by Ananthavijay** [comment]
@tachyons This can be closed since the issue has been fixed in another PR.

**Comment by tachyons** [comment]
Duplicate of #1032

**Pull request closed without merging**